moshi/src/main/java/com/squareup/moshi/JsonReader.java

```diff
@@ -528,6 +532,11 @@ private Options(String[] strings, okio.Options doubleQuoteSuffix) {
       this.doubleQuoteSuffix = doubleQuoteSuffix;
     }
 
+    /** Returns a copy of this {@link Options Option's} strings. */
+    public Set<String> getStrings() {
```

Comment:
should this be just `strings`? I don’t think we have a `get` prefix in Moshi anywhere else.

Reply:
my head's stuck in "make this nice for kotlin property syntax" land now, can change

Reply:
f6bdaec